app/src/main/java/org/supla/android/widget/lightswitch/LightSwitch.kt

```diff
@@ -0,0 +1,155 @@
+package org.supla.android.widget.lightswitch
```

Comment:
I'm not sure if it should be "LightSwitch" or "OnOffSwitch". If OnOff, SUPLA_CHANNELFNC_POWERSWITCH can be added to the function list. The icon should match the default icon assigned to the channel function. User uploaded icons should be included.